# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/doc/Debugger/Debugger.Frame.md
# Commit: 2828f072134e
# Full Hash: 2828f072134e0425521de5ad9d60ae880c57e944
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 1
# Description:
#   Bug 1600204 - Part 3: Support .constructing on suspended generator frames. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55233
# ==============================================================================

diff -r 73c33bf110d6 -r 2828f072134e js/src/doc/Debugger/Debugger.Frame.md
--- a/js/src/doc/Debugger/Debugger.Frame.md	Fri Dec 06 02:57:20 2019 +0000
+++ b/js/src/doc/Debugger/Debugger.Frame.md	Sun Dec 08 23:53:57 2019 +0000
@@ -265,7 +265,7 @@
 True if this frame is for a function called as a constructor, false
 otherwise.
 
-Accessing this property will throw if `.onStack == false`.
+Accessing this property will throw if `.terminated == true`.
 
 ### `arguments`
 The arguments passed to the current frame, or `null` if this is not a